src/ResourceManager/Sql/Commands.Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -0,0 +1,248 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Rest.Azure;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedInstance.Cmdlet
+{
+    /// <summary>
+    /// Defines the New-AzureSqlManagedInstance cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmSqlManagedInstance",
```

Comment:
You need to use the new pattern for cmdlet nouns:

```c#
 [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlManagedInstance"
```

Reply:
this is whyt the cmdlets are failing in NetCore

Reply:
Thanks Mark, this saved me a lot of time.
 
Do you happen to know if it is possible to have this for next deployment scheduled for end of this month (start of next month), if it gets merged soon?

Also, I see I have some Credscan issues due to which build fails, but those are not introduced with my code, but were there originally. What to do with it?